libraries/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -245,10 +241,9 @@ export abstract class BaseInstallAction extends BaseRushAction {
         for (const selectedSubspace of selectedSubspaces) {
           installManagerOptions.subspace = selectedSubspace;
           if (selectedSubspace.getPnpmOptions()?.alwaysFullInstall) {
-            installManagerOptions.pnpmFilterArguments = [];
+            installManagerOptions.selectedProjects = [];
```

Comment:
I think having an empty array makes sense for selecting all projects, perhaps a better name here would be `filteredProjects`?

Reply:
@william2958 then how would the list be provided to a subspace that contains none of the selected projects? Or does that become a non-issue because it would be filtered out earlier?

Reply:
@D4N14L when we are installing for a subspace, the array of affected projects for each subspace is pre-calculated in a map `selectedProjectsForSubspace` which is then passed to the BaseInstallManager. If you take a look at line 249 of BaseInstallAction, you will see where it loads up the selectedProjects for the current subspace

Reply:
Looking around/above in this file, only subspaces that are impacted will be in the `selectedSubspaces` list, and they must contain at least one project within that subspace that needs to be installed. So updating the property to `filteredProjects` feels more accurate then.

Reply:
Renamed to `filteredProjects`